content/_help/verify-your-identity/how-to-add-images-of-your-state-issued-id._fr.md

```diff
@@ -1,14 +1,115 @@
+Si vous utilisez un ordinateur pour vérifier votre identité, vous pourrez passer à un téléphone pour prendre des photos de votre pièce d'identité.
+
+### Conseils pour prendre des photos claires avec un téléphone
+
+<div class="grid-row grid-gap">
+  <div class="tablet:grid-col">
+    <img alt="La face de la pièce d'identité est centrée dans le cadre de la caméra et placée sur un fond sombre." src="{{ site.baseurl }}/assets/img/help/id-dos-and-donts/id-do-front.png" />
+  </div>
+  <div class="tablet:grid-col">
+    <img alt="Le dos de pièce d'identité est centré dans le cadre de la caméra et placé sur un fond sombre." src="{{ site.baseurl }}/assets/img/help/id-dos-and-donts/id-do-back.png" />
+  </div>
+</div>
+
+{%
+  include components/icon-list.html
+  items=page.do_list
+  size='md'
+  icon_color='green'
+  icon_shape='check_circle'
+%}
+
+### Problèmes courants liés à la prise de photos d'une pièce d'identité
+
+<div class="grid-row grid-gap margin-bottom-6">
+  <div class="tablet:grid-col">
+    {%
+      include components/image-example-incorrect.html
+      image-alt-text="La pièce d'identité est placée sur un fond blanc."
+      image='/assets/img/help/id-dos-and-donts/id-dont-white-background.png'
+      bordered_image=true
+      do_dont_text="À NE PAS FAIRE"
+      text='utiliser un fond blanc'
+    %}
+  </div>
+  <div class="tablet:grid-col">
+    {%
+      include components/image-example-incorrect.html
+      image-alt-text="La pièce d'identité est placée sur un fond texturé."
+      image='/assets/img/help/id-dos-and-donts/id-dont-textured-background.png'
+      bordered_image=true
+      do_dont_text="À NE PAS FAIRE"
+      text='utiliser des arrière-plans texturés à motifs'
+    %}
+  </div>
+</div>
+<div class="grid-row grid-gap">
+  <div class="tablet:grid-col">
+    {%
+      include components/image-example-incorrect.html
+      image-alt-text="Les bords de la pièce d'identité sont tronqués."
+      image='/assets/img/help/id-dos-and-donts/id-dont-cropped.png'
+      bordered_image=true
+      do_dont_text="À NE PAS FAIRE"
+      text="recadrer les bords de votre pièce d'identité"
+    %}
+  </div>
+  <div class="tablet:grid-col">
+    {%
+      include components/image-example-incorrect.html
+      image-alt-text="Il y a des reflets et des ombres projetées sur la pièce d'identité."
+      image='/assets/img/help/id-dos-and-donts/id-dont-shadow.png'
+      do_dont_text="À ÉVITER"
+      text='les reflets et les ombres'
+    %}
+  </div>
+</div>
+
+<div class="page-content__horizontal-rule"></div>
+## Si vous n'avez pas accès à un téléphone équipé d'une caméra, vous pouvez téléverser un fichier ou utiliser un scanner
+* Utilisez un appareil photo haute résolution (8 mégapixels ou plus).  La plupart des appareils photo des téléphones ou des tablettes fonctionneront, mais pas forcément la webcam de votre ordinateur. [Nous vous recommandons d'utiliser votre appareil mobile](#phone-tips).
 * Utilisez un fond solide et sombre.
```

Comment:
Lines 113 to 124 are the "Do" French translations. We want the French translations from live site's section called "Tips for uploading or scanning your ID:" (we are renaming this to "If you don’t have access to a phone with a camera, you can upload a file or use a scanner"). 

So we need to replace those bullets with these:
* Enregistrez chaque fichier au format JPG ou PNG. Vous ne pouvez pas utiliser un fichier PDF. Regardez le nom de votre fichier et il devrait avoir une abréviation après le titre (ex : JohnDoeID_Front.jpg) 
* Veillez à ce que vos images soient en haute résolution (environ 2025 x 1275 pixels)
* Les images doivent être en couleur

Reply:
I'm not sure I got this right, but I replaced the french bullets with these three ([commit](https://github.com/18F/identity-site/pull/1151/commits/60f1736fb1d2a533542c2ae7da5f0a4b9f3f2741)) like this:

<img width="857" alt="Screenshot 2023-09-07 at 13 30 31" src="https://github.com/18F/identity-site/assets/6818839/d8a07b28-5783-44f6-b00c-03d97ae0a903">

Reply:
I left a new comment with the full set of French translations, sorry for the confusion! Let me know if I can clarify anything